File: appgraph.cpp
#include <iostream>
#include "graph.h"

using namespace std;

int main()
{
    Graph g("exemplos/tinyG.txt");
    // g.addEdge("0", "1");
    // g.addEdge("0", "2");
    // g.addEdge("2", "1");

    for (auto const& v : g.getVerts()) {
      cout << v << ": ";
      for (auto const& w : g.getAdj(v))
        cout << w << " ";
      cout << endl;
	}

  cout << endl;
  cout << g.toDot();
}

File: AppGraph.java
public class AppGraph
{
  public static void main(String[] args) {
    /*
    Graph g = new Graph();
    g.addEdge("0", "1");
    g.addEdge("0", "2");
    g.addEdge("2", "1");
    */
    Graph g = new Graph("exemplos/tinyG.txt");

    for (String v : g.getVerts()) {
      System.out.print(v + ": ");
      for (String w : g.getAdj(v))
        System.out.print(w + " ");
      System.out.println();
    }

    System.out.println();
    System.out.println(g.toDot());
  }
}
